app/controllers/adopter_applications_controller.rb

```diff
@@ -1,38 +1,42 @@
 class AdopterApplicationsController < ApplicationController
-  before_action :authenticate_user!, :adopter_with_profile
-  before_action :check_pet_app_status, only: [:create]
+  verify_authorized
+
+  before_action :authenticate_user!
 
   def index
-    @applications = AdopterApplication.where(adopter_account_id:
-                                             current_user.adopter_account.id)
+    authorize!
+
+    @applications = authorized_scope(AdopterApplication.all)
```

Comment:
so this `authorized_scope` is?

```
relation_scope do |relation|
    relation.where(organization: user.organization)
  end
```

Reply:
So this will allow an adopter to see all adopter applications scoped under the org. This should also restrict applications to show only those made by the adopter. They should not see any one elses.

Reply:
You are correct. `relation_scope` is the 'default scope' that is used for `authorized_scope` for Rails with Action Policy. (More detail: The "relation" in `relation_scope` actually refers to `ActiveRecord::Relation`. When `authorized_scope` is called with an `ActiveRecord::Relation`, it will match with this `relation_scope` in the policy.)

Sorry again for the confusion. This is also related to the next commit as this [comment](https://github.com/rubyforgood/pet-rescue/pull/466#discussion_r1508533397) mentions. In that commit, I am overwriting the `relation_scope` defined in `ApplicationPolicy` by redefining it in `AdopterApplicationPolicy`.

At some commit, I did end up adding this comment to help clarify this:
```ruby
  # Default authorized_scope; override for individual policies if necessary.
```

In regards to the default in `ApplicationPolicy`:
```ruby
  relation_scope do |relation|
    relation.where(organization: user.organization)
  end
```
I thought this would be a good default, since I don't think there is any action where we would want a user to be able to scope outside of their own org.

Reply:
Yes I agree that is a good default.